pubsub/rabbitpubsub/doc.go

```diff
@@ -23,7 +25,7 @@
 // A Pub/Sub subscription is an AMQP queue. The queue should be bound to the exchange
 // that is the topic of the subscription. See the package example for details.
 //
-// This package exposes the following types for As:
-// Topic: *amqp.Connection
-// Subscription: *amqp.Connection
+// As
```

Comment:
This should probably be replaced with a complete sentence, like "The As method can be called on the following generic types to get the following provider-specific types."

Reply:
There's a full example of how to use `As` on `Topic.As`.